From 18483fc7b09191376a28b7e5f2a23642daae9da7 Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Mon, 13 Nov 2017 22:11:01 -0600
Subject: [PATCH] fix win zlib and openssl lib names

---
 src/tools/msvc/Solution.pm | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/tools/msvc/Solution.pm b/src/tools/msvc/Solution.pm
index a8fdc31..3880afe 100644
--- a/src/tools/msvc/Solution.pm
+++ b/src/tools/msvc/Solution.pm
@@ -518,24 +518,24 @@ sub AddProject
 	if ($self->{options}->{zlib})
 	{
 		$proj->AddIncludeDir($self->{options}->{zlib} . '\include');
-		$proj->AddLibrary($self->{options}->{zlib} . '\lib\zdll.lib');
+		$proj->AddLibrary($self->{options}->{zlib} . '\lib\zlib.lib');
 	}
 	if ($self->{options}->{openssl})
 	{
 		$proj->AddIncludeDir($self->{options}->{openssl} . '\include');
-		if (-e "$self->{options}->{openssl}/lib/VC/ssleay32MD.lib")
+		if (-e "$self->{options}->{openssl}/lib/ssleay32.lib")
 		{
 			$proj->AddLibrary(
-				$self->{options}->{openssl} . '\lib\VC\ssleay32.lib', 1);
+				$self->{options}->{openssl} . '\lib\ssleay32.lib');
 			$proj->AddLibrary(
-				$self->{options}->{openssl} . '\lib\VC\libeay32.lib', 1);
+				$self->{options}->{openssl} . '\lib\libeay32.lib');
 		}
 		else
 		{
 			$proj->AddLibrary(
-				$self->{options}->{openssl} . '\lib\ssleay32.lib', 1);
+				$self->{options}->{openssl} . '\lib\ssleay32.lib');
 			$proj->AddLibrary(
-				$self->{options}->{openssl} . '\lib\libeay32.lib', 1);
+				$self->{options}->{openssl} . '\lib\libeay32.lib');
 		}
 	}
 	if ($self->{options}->{nls})
-- 
2.11.1

