--- postgresql-13.1.orig/src/tools/msvc/Solution.pm	2020-11-09 19:24:30.000000000 -0300
+++ postgresql-13.1/src/tools/msvc/Solution.pm	2021-01-25 17:36:45.877508849 -0300
@@ -990,9 +990,9 @@
 				# to be here, so don't ask for it in last
 				# parameter.
 				$proj->AddLibrary(
-					$self->{options}->{openssl} . '\lib\ssleay32.lib', 0);
+					$self->{options}->{openssl} . '\lib\libcrypto.lib', 0);
 				$proj->AddLibrary(
-					$self->{options}->{openssl} . '\lib\libeay32.lib', 0);
+					$self->{options}->{openssl} . '\lib\libssl.lib', 0);
 			}
 		}
 	}
