diff --git a/src/tools/msvc/Solution.pm b/src/tools/msvc/Solution.pm
index 9ba3acf..3c7d3c5 100644
--- a/src/tools/msvc/Solution.pm
+++ b/src/tools/msvc/Solution.pm
@@ -1006,7 +1006,6 @@
 	if ($self->{options}->{gss})
 	{
 		$proj->AddIncludeDir($self->{options}->{gss} . '\include');
-		$proj->AddIncludeDir($self->{options}->{gss} . '\include\krb5');
 		if ($self->{platform} eq 'Win32')
 		{
 			$proj->AddLibrary(
@@ -1019,11 +1018,11 @@
 		else
 		{
 			$proj->AddLibrary(
-				$self->{options}->{gss} . '\lib\amd64\krb5_64.lib');
+				$self->{options}->{gss} . '\lib\krb5_64.lib');
 			$proj->AddLibrary(
-				$self->{options}->{gss} . '\lib\amd64\comerr64.lib');
+				$self->{options}->{gss} . '\lib\comerr64.lib');
 			$proj->AddLibrary(
-				$self->{options}->{gss} . '\lib\amd64\gssapi64.lib');
+				$self->{options}->{gss} . '\lib\gssapi64.lib');
 		}
 	}
 	if ($self->{options}->{iconv})
