diff --git a/src/tools/msvc/Solution.pm b/src/tools/msvc/Solution.pm
index 9ba3acf..3c7d3c5 100644
--- a/src/tools/msvc/Solution.pm
+++ b/src/tools/msvc/Solution.pm
@@ -548,10 +548,10 @@ sub AddProject
 	}
 	if ($self->{options}->{gss})
 	{
-		$proj->AddIncludeDir($self->{options}->{gss} . '\inc\krb5');
-		$proj->AddLibrary($self->{options}->{gss} . '\lib\i386\krb5_32.lib');
-		$proj->AddLibrary($self->{options}->{gss} . '\lib\i386\comerr32.lib');
-		$proj->AddLibrary($self->{options}->{gss} . '\lib\i386\gssapi32.lib');
+		$proj->AddIncludeDir($self->{options}->{gss} . '\include');
+		$proj->AddLibrary($self->{options}->{gss} . '\lib\krb5_64.lib');
+		$proj->AddLibrary($self->{options}->{gss} . '\lib\comerr64.lib');
+		$proj->AddLibrary($self->{options}->{gss} . '\lib\gssapi64.lib');
 	}
 	if ($self->{options}->{iconv})
 	{
